tests/cases/fourslash/autoImportSortCaseSensitivity1.ts

```diff
@@ -28,7 +28,7 @@
 //// d/*1*/
 
 goTo.marker("0");
-verify.importFixAtPosition([`import { A, B, C, a } from "./exports1";\na`]);
+verify.importFixAtPosition([`import { a, A, B, C } from "./exports1";\na`]);
```

Comment:
I assume since this is ambiguous, it doesn't really matter what we choose here?

Reply:
For this particular test, since `ignoreCase = true`, `a` and `A` should be together, the test was wrong before 🤔

Reply:
Oh, didn't notice that ignore case was on!